hydradx/spec/OmnipoolSpec.ipynb

```diff
@@ -66,8 +65,6 @@
     "| $p_i^j$ | Price of asset $i$ denominated in $j$ | $\\frac{Q_i}{Q_j}\\frac{R_j}{R_i}$ |\n",
     "| $W_i$ | Weight of asset $i$ in pool | $\\frac{Q_i}{Q}$                  |\n",
     "| $Y_i$ | Swap invariant of asset $i$ in pool | $Q_i R_i$                        |\n",
-    "| $T$ | Approximate TVL in USD in Omnipool | $R_U\\frac{Q}{Q_U}$               |\n",
-    "| $T_i$ | Approximate TVL in USD in asset $i$ | $R_U\\frac{Q_i}{Q_U}$             |\n",
     "\n",
     "Note that we assume here that the preffered stablecoin is asset $U$."
```

Comment:
Should probably get rid of this line too

Reply:
good catch